pandas/core/internals/blocks.py

```diff
@@ -1828,10 +1828,9 @@ def interpolate(self, method='pad', axis=0, inplace=False, limit=None,
             placement=self.mgr_locs)
 
     def shift(self,
-              periods,                  # type: int
-              axis=0,                   # type: libinternals.BlockPlacement
-              fill_value=None):         # type: Any
-        # type: (...) -> List[ExtensionBlock]
+              periods: int,
+              axis: libinternals.BlockPlacement = 0,
+              fill_value: Optional[Any] = None) -> List['ExtensionBlock']:
```

Comment:
Is Any not inclusive of Optional already?

Reply:
Good catch.